seed/static/seed/js/controllers/inventory_map_controller.js

```diff
@@ -0,0 +1,409 @@
+/**
+ * :copyright (c) 2014 - 2018, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
+ * :author
+ */
+angular.module('BE.seed.controller.inventory_map', [])
+  .controller('inventory_map_controller', [
+    '$scope',
+    '$stateParams',
+    '$state',
+    'cycles',
+    'inventory',
+    'inventory_service',
+    'labels',
+    'urls',
+    'spinner_utility',
+    function ($scope,
+              $stateParams,
+              $state,
+              cycles,
+              inventory,
+              inventory_service,
+              labels,
+              urls,
+              spinner_utility) {
+      spinner_utility.show();
+
+      $scope.inventory_type = $stateParams.inventory_type;
+      $scope.data = inventory.results;
+
+      $scope.geocoded_data = _.filter($scope.data, function(building) {
+        return building.long_lat
+      });
+      $scope.ungeocoded_data = _.filter($scope.data, function(building) {
+        return !building.long_lat
+      });
+
+      // Map
+      var base_layer = new ol.layer.Tile({
+        source: new ol.source.Stamen({
+          layer: 'terrain'
+        }),
+        zIndex: 0,  // Note: This is used for layer toggling.
+      });
+
+      // Define buildings source - the basis of layers
+      var buildingPoint = function (building) {
+        var format = new ol.format.WKT();
+
+        var long_lat = building.long_lat
+        var feature = format.readFeature(long_lat, {
+          dataProjection: 'EPSG:4326',
+          featureProjection: 'EPSG:3857'
+        });
+
+        feature.setProperties(building)
+        return feature
+      };
+
+      var buildingSources = function (records = $scope.geocoded_data) {
+        var features = _.map(records, buildingPoint);
+
+        return new ol.source.Vector({ features: features });
+      };
+
+      // Points/clusters layer
+      var clusterPointStyle = function (size) {
+        // TODO revisit this with a larger dataset
```

Comment:
I'd like to revisit the size of the cluster points after seeing other/different datasets.